src/SDKs/Relay/Management.Relay/Generated/Models/AccessKeys.cs

```diff
@@ -2,30 +2,33 @@
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 //
-// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Code generated by Microsoft (R) AutoRest Code Generator 1.1.0.0
 // Changes may cause incorrect behavior and will be lost if the code is
 // regenerated.
 
 namespace Microsoft.Azure.Management.Relay.Models
 {
-    using Azure;
-    using Management;
-    using Relay;
+    using Microsoft.Azure;
+    using Microsoft.Azure.Management;
+    using Microsoft.Azure.Management.Relay;
     using Newtonsoft.Json;
     using System.Linq;
 
     /// <summary>
     /// Namespace/Relay Connection String
     /// </summary>
-    public partial class AuthorizationRuleKeys
+    public partial class AccessKeys
```

Comment:
@v-Ajnava will this change have any impact on PowerShell due to change in type?
Please plan accordingly, you cannot introduce breaking changes in PS